pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -406,11 +487,23 @@ public static void main(String[] args) throws Exception {
             long total = totalMessagesReceived.sum();
             double rate = messagesReceived.sumThenReset() / elapsed;
             double throughput = bytesReceived.sumThenReset() / elapsed * 8 / 1024 / 1024;
-
+            double rateAck = messageAck.sumThenReset() / elapsed;
+            long totalTransaction = 0;
+            double averageTimePerTransaction = 0;
+            if (arguments.isEnableTransaction) {
+                totalTransaction = totalNumTransaction.sum();
+                averageTimePerTransaction = elapsed / numTransaction.sumThenReset();
+            }
             reportHistogram = recorder.getIntervalHistogram(reportHistogram);
 
+            String transactionLog = arguments.isEnableTransaction ? "---transaction: " + totalTransaction +
```

Comment:
emm~, If  totalEndTxnOpFailNum and totalEndTxnOpSuccessNum print in printAggregatedThroughput can be better?This is an instant data, not a conclusive data. The number of failed commit transactions should be relatively small, compared to printing the number of failures within this time at a fixed time, is it more valuable to print the last one?

Reply:
It is very enlightening. Printing the total number in each log is more helpful for users to grasp the overall situation, instead of waiting for the end of each time to know the overall situation of the performance test. In fact, as long as there is no conflict, we should put more information on it